src/js/providers/flash.js

```diff
@@ -222,7 +222,6 @@ define([
                         this.sendEvent(events.JWPLAYER_MEDIA_ERROR, {
                             message: 'Error loading media: File could not be played'
                         });
-                        this.setState(states.IDLE);
```

Comment:
Why not use 

    this.setState(states.ERROR);  //?

Without it, the mediaModel's state and the models state will fall out of sync